source/usage-examples/command.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. Call the ``runCommand()`` method
+with your command object on an instance of a database for diagnostic and
+administrative tasks such as fetching server stats or initializing a
+replica set.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java Driver whenever possible.
+
+The ``runCommand()`` method accepts a :manual:`command
+<reference/command/>` in Bson format. By default, ``runCommand`` returns
+an object of type ``org.bson.Document`` containing the output of the
+database command.
+
+You can specify a :manual:`read preference <core/read-preference/>` or
+a return type for ``runCommand()`` as an optional second parameter. To
+specify both a read preference and a return type, pass the read
+preference as a second parameter and the return type as a third
+parameter.
+
+Example
+-------
+
+.. include:: /includes/connect-guide-note.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/Command.java
+  :language: java
+  :linenos:
```

Comment:
I know some team is working on the "linenos" directive, but for now let's omit this.